frontend/src/actions/cartAction.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { CART_ADD_ITEM, CART_REMOVE_ITEM } from "../constants/cartConstants.js"
+
+
+export const addToCart = (id, qty) => async (dispatch, getState) => {
+    const { data } = await axios.get(`/api/products/${id}`);
+
+
+    await axios.get(`/api/products`)
+
+    dispatch({
+        type: CART_ADD_ITEM, payload: {
```

Comment:
Codacy has a fix for the issue: [Replace `········type:·CART_ADD_ITEM,` with `····type:·CART_ADD_ITEM,⏎···`](https://app.codacy.com/gh/Durotemy/e-commerce/pullRequest?prid=9481957)
```suggestion
    type: CART_ADD_ITEM,
```